tslint.json

```diff
@@ -2,7 +2,6 @@
   "extends": "tslint:recommended",
   "rules": {
     "align": false,
-    "ban-types": false,
```

Comment:
The TSLint [Rule: ban-types](https://palantir.github.io/tslint/rules/ban-types/) says the `ban-types` option should be a list of regex/explanation pairs (wow, powerful concept) but doesn't mention any default value.    It appears there is an (undocumented?) default banning `Symbol` and `Object`.   I guess those make sense, have you found anything documenting why those in particular are banned?

Reply:
I tried changing a couple of these back for fun.   Excellent how VSCode is catching them virtually immediately (with TSLint add-in enabled.)

Reply:
The default is supplied by the recommended rules from which our configuration is derived.